exporters/readers/s3_reader.py

```diff
@@ -61,7 +61,7 @@ def format_prefixes(prefixes, start, end):
 
 @retry_short
 def read_chunk(key):
-    return key.read(1024 * 8)
+    return key.read(1024)
```

Comment:
Ah, I see it _was_ intended `Chunk size was provoking decompressors objects to behave incorrectly.`
So, what was happening with the bigger chunk size that won't happen with the smaller one?